cli/cmd/auth.go

```diff
@@ -16,6 +19,12 @@ func NewLoginCmd(injector do.Injector) *cobra.Command {
 		Short: fmt.Sprintf("Login to %s", version.ProductName),
 		Long:  fmt.Sprintf("Login to the %s CLI.", version.ProductName),
 		Run: func(cmd *cobra.Command, args []string) {
+			if os.Getenv(AccessTokenEnvVar) != "" {
+				fmt.Printf("\n%s Already authenticated using a Personal Access Token.\n", style.Yellow(icons.Info))
```

Comment:
I'd say configured rather than authenticated, we know that a PAT was provided but we don't know if it's valid at this point.

Reply:
Yup thats a good idea - I'm going to pull this PR, I'll add this note to the parent issue and come back to it on Friday.